pkg/client/unversioned/clientcmd/loader.go

```diff
@@ -148,6 +149,29 @@ func NewDefaultClientConfigLoadingRules() *ClientConfigLoadingRules {
 	}
 }
 
+func loadConfigDir(dir string) []string {
+	stat, err := os.Lstat(RecommendedHomeFileDir)
+	if err != nil {
+		if !os.IsNotExist(err) {
+			glog.Warningf("Failed to stat config dir: %v", err)
+		}
+		return []string{}
+	}
+	if !stat.IsDir() {
+		glog.Warningf("Expected %s to be a directory", dir)
```

Comment:
should we `return []string{}` now - I guess otherwise we'll print another error?